apollo-router/src/plugins/telemetry/metrics/layer.rs

```diff
@@ -26,7 +26,7 @@ use super::METRIC_PREFIX_VALUE;
 
 macro_rules! log_and_panic_in_debug_build {
     ($($tokens:tt)+) => {{
-        tracing::debug!($($tokens)+);
+        tracing::error!($($tokens)+);
```

Comment:
Note that this does mean there is the potential to spam logs if they have their attributes incorrect.

Reply:
Yes, but for now it's completely invisible. I'm tempted to put this either as a warn or error but it should be louder than today because it's discarded. What do you think ?